nixos/modules/services/mail/dovecot.nix

```
@@ -473,6 +468,17 @@ in
         description = "Programs available for use by the vnd.dovecot.pipe extension";
         type = types.listOf types.path;
       };
+
+      extraPluginSettings = mkOption {
+        type = types.attrsOf types.anything;
```

```suggestion
        type = types.attrsOf types.str;
```

Right now, this will only work for values that can be interpolated, so strings and attrsets with toString/outPath. Not sure if `str` covers those.

---

Done